migration/src/m20230130_124740_read_only_login_logic.rs

```diff
@@ -0,0 +1,43 @@
+use sea_orm_migration::prelude::*;
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        // Replace the sample below with your own migration scripts
+        manager
+            .alter_table(
+                Table::alter()
+                    .table(Alias::new("login"))
```

Comment:
Nitpick: this should be Admin::Table. Doesn't make a huge difference but I try to only use Alias when I don't have an Iden enum available